src/components/Lecturers/Lecturers.elements.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from "styled-components"
+import { colors } from '../../styled/variables'
+
+export const CardsContainer = styled.div`
+    display: flex;
+    justify-content: center;
+    margin: 0 auto 30px auto;
+    flex-wrap: wrap;
+`
+
+export const Card = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 285px;
```

Comment:
Vím, že takhle to bylo zadaný v designu :) Mně se zpravidla vyplatilo přesným rozměrům v `px` vyhnout a tvořit v procentech. Hlavně u `height` to bývá potíž, ta mi přijde se uvádí málokdy. Ale je to částečně i otázka stylu a toho, jak je kdo zvyklý kódovat :)